plugins/processors/strings/README.md

```diff
@@ -78,6 +79,11 @@ If you'd like to apply multiple processings to the same `tag_key` or `field_key`
   ## Decode a base64 encoded utf-8 string
   # [[processors.strings.base64decode]]
   #   field = "message"
+
+  ## Sanitize a string to ensure it is a valid utf-8 string. Non-valid characters are replaced by 'new'
```

Comment:
Could you clarify your description to align more closely with the go documentation around ToValidUTF8 function docs? Here's a link to the relevant documentation and description: https://golang.org/pkg/strings/#ToValidUTF8